action.yml

```diff
@@ -1,22 +1,44 @@
-name: 'Sourcery GitHub Action'
+name: "Sourcery GitHub Action"
 
-description: 'A GitHub Action for running Sourcery.'
+branding:
+  icon: 'box'
+  color: 'orange'
+
+description: "A GitHub Action for running Sourcery."
 
 inputs:
   token:
-    description: 'The Sourcery token for logging in.'
+    description: "The Sourcery token for logging in."
     required: true
-  options:
-    description:
-      "Options passed to Sourcery. Run `sourcery review --help` to see available options."
+  target:
+    description: "File(s) or directory(ies) to review."
     required: false
-    default: "--check"
-  base_ref:
-    description: "Reference to compare the current branch with and run Sourcery on the diff. If empty, run Sourcery on the current branch."
+    default: "."
+  version:
+    description: "Sourcery CLI version, e.g. '0.12.10'. If empty, default to the latest version."
```

Comment:
I wonder if this version number 0.12.10 can be automatically updated on each release?

Reply:
@brendanator I quickly looked into it, but there doesn't seem to be an immediate way to do that (do you know any?)

For now, I added [a card to our product backlog](https://www.notion.so/sourcery/Automatically-update-suggested-Sourcery-version-in-the-GitHub-action-744df660397f427bb97a6792f562ea84) for us to keep track of this possible feature.